kvscheduler/descriptor/adapter-generator/template.go

```diff
@@ -0,0 +1,353 @@
+package main
+
+const Template = `
+// Code generated by adapter-generator. DO NOT EDIT.
+// This file was generated by at {{ .Timestamp }}
```

Comment:
Generated file should not contain timestamp, otherwise it will be harder to do checks if the generated file changed. The file itself has modification timestamp.